src/mrack/transformers/transformer.py

```diff
@@ -15,8 +15,13 @@
 """Generic Transformer."""
 
 import logging
+import os
+import sys
 import typing
 
+import gssapi
```

Comment:
IMO should be optional dependency as will be used in specific runtime env. 
So for this we will be modify setup.py with extras_require?

Reply:
I think for now we should keep it in requirements.txt. Later when we will update the python package to have subpackages we can think of this. I will link above comment to issue I have created so we do not forget.

Reply:
https://github.com/neoave/mrack/issues/213#issuecomment-1309915790

Reply:
There is also rpm packaging part. Bigger question is what package should depend on this. The way it is written indicates that python3-mracklib should get a new dependency and thus it would need to be reflected in mrack.spec.  Alternative way is to make it optional and only recommend it in mrack.spec.